app/src/main/java/com/h/pixeldroid/api/PixelfedAPI.kt

```diff
@@ -1,12 +1,18 @@
 package com.h.pixeldroid.api
 
-import com.h.pixeldroid.objects.*
+import com.h.pixeldroid.objects.Application
+import com.h.pixeldroid.objects.Status
+import com.h.pixeldroid.objects.Token
 import retrofit2.Call
 import retrofit2.Retrofit
 import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
 import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.FormUrlEncoded
 import retrofit2.http.GET
+import retrofit2.http.POST
 import retrofit2.http.Query
+import retrofit2.http.Field
```

Comment:
c'est peut-être worth-it de faire un `import retrofit2.http.*` ici

Reply:
Comme est spécifié dans le style guide, il est proscrit d'avoir des imports à wildcard.

Reply:
ok